src/sources/http.rs

```diff
@@ -43,6 +43,8 @@ pub(super) struct SimpleHttpConfig {
     path: String,
     #[serde(default = "default_path_key")]
     path_key: String,
+    #[serde(default = "default_http_method")]
+    http_method: String,
```

Comment:
I think it'd be best to define an enum for this to deserialize into so that users get a configuration error if they use an unsupported `http_method`.